crates/crypto/core/src/lib.rs

```diff
@@ -7,8 +7,10 @@ use serde::{Deserialize, Serialize};
 pub enum KeyTypeId {
     #[cfg(feature = "bn254")]
     Bn254,
-    #[cfg(any(feature = "k256", feature = "tangle"))]
+    #[cfg(feature = "tangle")]
     Ecdsa,
+    #[cfg(feature = "k256")]
+    K256,
```

Comment:
Why add a new key type id? `KeyTypeId::Ecdsa` covers both the `SpEcdsa` and `K256Ecdsa` types.